packages/core/strapi/src/cli/commands/cloud/config/local.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import os from 'os';
+import fs from 'fs';
+
+export const TMP_DIR = path.join(os.tmpdir(), 'strapi-cli');
```

Comment:
Tmp dirs are made to be cleaned up and deleted without any regards to what's in there, you won't really want this to persist connection for the lifetime of the token, the homedir will be more appropriate.

Reply:
Aren't tokens meant to be temporary? 🤔
I doubt that users want the remaining tokens in their files to live there indefinitely for no reason. (at least I would not want that as a user 😅 )

Reply:
It def should be temporary, The token has an expiration date I hope !
But it doesn't mean it should be deleted as soon as your computer decides to cleanup the tmp folder. You should stay connected for the duration of that token unless you decide to logout or delete the cache file. 

This is the standard practice and there is even a pkg (configstore) that a lot of lib use to store those tokens. 

I wouldn't want to be logged out of my github cli everytime I restart my computer (of course the tmp dir isn't cleared than often but you have no guarantees)

Reply:
I just answered at the same time :)